packages/c/cryptopp/xmake.lua

```diff
@@ -17,7 +19,20 @@ package("cryptopp")
     on_install("windows", "macosx", "linux", "bsd", "iphoneos", function (package)
         local cryptopp_cmake = package:resourcedir("cryptopp_cmake")
         os.cp(path.join(cryptopp_cmake, "*", "CMakeLists.txt"), ".")
-        os.cp(path.join(cryptopp_cmake, "*", "cryptopp-config.cmake"), ".")
+        local version = package:version()
+        
+        if (version:ge("8.7")) then
+            if (package:config("shared")) then
+                print("ERROR: Crypto++ 8.7+ does not support shared libraries")
+                return
```

Comment:
you can disable shared, see

https://github.com/xmake-io/xmake-repo/blob/766154c7806aca96c87500f2e84b70c9dd1e1b5b/packages/m/mma/xmake.lua#L11

Reply:
Thank you, I'll make this fix tomorrow.